game-using-jsDOM/script.js

```diff
@@ -0,0 +1,217 @@
+// Array of words that we will use for the game 
+const words = ['javascript', 'penguins', 'elephant', 'koala', 'synonym', 'peanut',
```

Comment:
lol, if i were not a programmer i'd be so mad at a word guessing game that has me guess "javascript" 💀

Reply:
Oops hehe I totally forgot to remove those words since I was just thinking of some at the top of my head. I have updated the array to have more words and removed javascript!